src/assets/scss/base/_manifest.scss

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Why extra lines in multiple files. You can write detail about `scss` in the comment section.